tests/dbus/test_dbus_client.cpp

```diff
@@ -261,6 +261,19 @@ void CheckNat64(ThreadApiDBus *aApi)
 #endif
 }
 
+void CheckEphemeralKey(ThreadApiDBus *aApi)
+{
+    OTBR_UNUSED_VARIABLE(aApi);
+    bool enabled;
+    bool original;
+
+    TEST_ASSERT(aApi->GetEphemeralKeyEnabled(original) == OTBR_ERROR_NONE);
+    TEST_ASSERT(aApi->SetEphemeralKeyEnabled(!original) == OTBR_ERROR_NONE);
+    TEST_ASSERT(aApi->GetEphemeralKeyEnabled(enabled) == OTBR_ERROR_NONE);
+    TEST_ASSERT(enabled == !original);
+    TEST_ASSERT(aApi->GetEphemeralKeyEnabled(original) == OTBR_ERROR_NONE);
```

Comment:
As mentioned earlier in a different PR, please use a more deterministic routine. For example, set it to false at the beginning and set it to true later.

Reply:
As there are no dependencies on the ePSKc original state, it should be OK to use deterministic way. Thanks~